README.md

```diff
@@ -1 +1,33 @@
-# drupal-nginx-fpm
\ No newline at end of file
+# Drupal-nginx-php Docker 
+This is a Drupal Docker image which can run on both 
+ - [Azure Web App on Linux](https://docs.microsoft.com/en-us/azure/app-service-web/app-service-linux-intro) 
+ - [Drupal on Linux Web App With MySQL](https://ms.portal.azure.com/#create/Drupal.Drupalonlinux )
+ - Your Docker engines's host.
```

Comment:
Usually running on a Docker engine's host needs VOLUME feature. But we don't enable that here. So we can remove this statement. Running on Azure should be our TOP and FIRST target.

Reply:
The same line also exist with wordpress.
![image](https://user-images.githubusercontent.com/14034268/34753897-7c900900-f56f-11e7-9c67-d7f776f03552.png)

i think they can use "-v" parameters if they are using their Docker engine's host. I also use it when  during debug/run in my ubuntu machine.

Just avoid it with Azure web app, and looks they have no way to add VOLUME during deploy to azure.